<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tic Tac Toe</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="ShapeLabel" name="cell_0">
      <property name="styleSheet">
       <string notr="true">background-color: #282828;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="ShapeLabel" name="cell_1">
      <property name="styleSheet">
       <string notr="true">background-color: #282828;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="ShapeLabel" name="cell_2">
      <property name="styleSheet">
       <string notr="true">background-color: #282828;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="ShapeLabel" name="cell_3">
      <property name="styleSheet">
       <string notr="true">background-color: #282828;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="ShapeLabel" name="cell_4">
      <property name="styleSheet">
       <string notr="true">background-color: #282828;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="ShapeLabel" name="cell_8">
      <property name="styleSheet">
       <string notr="true">background-color: #282828;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="ShapeLabel" name="cell_5">
      <property name="styleSheet">
       <string notr="true">background-color: #282828;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="ShapeLabel" name="cell_6">
      <property name="styleSheet">
       <string notr="true">background-color: #282828;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="ShapeLabel" name="cell_7">
      <property name="styleSheet">
       <string notr="true">background-color: #282828;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnections">
    <property name="title">
     <string>Connections</string>
    </property>
    <addaction name="actionSetAsServer"/>
    <addaction name="actionSetAsClient"/>
    <addaction name="actionConnect"/>
    <addaction name="actionInfo"/>
   </widget>
   <addaction name="menuConnections"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSetAsServer">
   <property name="text">
    <string>Set as server</string>
   </property>
  </action>
  <action name="actionSetAsClient">
   <property name="text">
    <string>Set as client</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>Info</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ShapeLabel</class>
   <extends>QLabel</extends>
   <header>shape_label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
